src/router.ts

```diff
@@ -443,6 +443,27 @@ async function introduceNewVoterToSlackChannel(
   return response.data.ts;
 }
 
+export async function parseStateKeyword(
+  redisClient: PromisifiedRedisClient,
+  userInfo: UserInfo,
+  userMessage: string
+): Promise<boolean> {
+  // Is this a special keyword?
+  const keywords = await RedisApiUtil.getHash(redisClient, 'keywordConfig');
```

Comment:
Do we want this in redis? I think this is unlikely to change frequently, so it feels like an env var or a configuration in the code based on CLIENT_ORGANIZATION would be easier to manage.

Reply:
Yeah, that's simpler.  Updated